.github/workflows/run-dangerjs.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           MONITORED_PATHS=(
             'front/lib/models/'
             'front/lib/resources/storage/models/'
+            'front/lib/registry.ts'
```

Comment:
why adding it here ? we already have checks on registry, outside of danger ?

Reply:
Feedback from @flvndvd to get the runbook popup in comments. I agreed with on the more the merrier